### ISSUE TITLE
gopanda url outdated

### ISSUE THREAD

**Issue by danielb2** [caskroom/homebrew-cask#17673]
Should read: http://pandanet-igs.com/gopanda2/installer/stable/osx-32/gopanda2-osx-32.zip

Note: used to be `mac-32` and now reads `osx-32`

FYI:
Attempted to follow instructions in [updating a cask](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask). Despite saying submitted I didn't notice any new PR made

``` bash
[~/l/D/L/T/c/h/Casks]$ cask-repair --pull origin --push danielb2 gopanda
----------------------------------------------------------------------------------------------------
cask 'gopanda' do
  version :latest
  sha256 :no_check

  url 'http://pandanet-igs.com/gopanda2/installer/stable/mac-32/gopanda2-mac-32.zip'
  name 'GoPanda'
  homepage 'http://pandanet-igs.com/communities/gopanda2'
  license :unknown # TODO: change license and remove this comment; ':unknown' is a machine-generated
 placeholder

  app 'GoPanda2.app'
end
----------------------------------------------------------------------------------------------------
Type the new version (or leave blank to abort)
> latest
Paste the new URL (or leave blank to use the current one)
> http://pandanet-igs.com/gopanda2/installer/stable/osx-32/gopanda2-osx-32.zip
diff --git a/Casks/gopanda.rb b/Casks/gopanda.rb
index a29f9b3..583e2a8 100644
--- a/Casks/gopanda.rb
+++ b/Casks/gopanda.rb
@@ -2,7 +2,7 @@ cask 'gopanda' do
   version :latest
   sha256 :no_check

-  url 'http://pandanet-igs.com/gopanda2/installer/stable/mac-32/gopanda2-mac-32.zip'
+  url 'http://pandanet-igs.com/gopanda2/installer/stable/osx-32/gopanda2-osx-32.zip'
   name 'GoPanda'
   homepage 'http://pandanet-igs.com/communities/gopanda2'
   license :unknown # TODO: change license and remove this comment; ':unknown' is a machine-generate
d placeholder
----------------------------------------------------------------------------------------------------
Is everything correct? You can also make further manual edits (y/n/e) y
fatal: 'danielb2' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Error creating pull request: Unprocessable Entity (HTTP 422)
Missing field: "head_sha"
Missing field: "base_sha"
No commits between master and update-gopanda
Error creating pull request: Unprocessable Entity (HTTP 422)
Missing field: "head_sha"
Missing field: "base_sha"
No commits between master and update-gopanda


Submitted ()
```

**Comment by victorpopkov** [comment]
The download URL used in the cask seems to be working. Of course, if you still wish to update the URL you can submit a PR, but there is no actual necessity in this at the moment.

The main reason why `cask-repair` script didn't work most likely is because you didn't fork the [homebrew-cask](https://github.com/caskroom/homebrew-cask) repository.

**Comment by vitorgalvao** [comment]
```bash
fatal: 'danielb2' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Error creating pull request: Unprocessable Entity (HTTP 422)
Missing field: "head_sha"
Missing field: "base_sha"
No commits between master and update-gopanda
Error creating pull request: Unprocessable Entity (HTTP 422)
Missing field: "head_sha"
Missing field: "base_sha"
No commits between master and update-gopanda
```

The `fatal` errors are the clue something went wrong. When it submits correctly, you’ll see the URL of the PR inside the parenthesis.

That isn’t a good experience, though, and I can see why you got confused. Will fix it now.

**Comment by vitorgalvao** [comment]
Fixed now in `cask-repair`, the message should be clearer.

**Comment by vitorgalvao** [comment]
Fixed `gopanda` in https://github.com/caskroom/homebrew-cask/pull/17678.

**Comment by danielb2** [comment]
@vitorgalvao the update instructions were to use: `cask-repair --pull origin --push $github_user $outdated_cask
`

So I replaced $github_user with my username.  Did I screw up, or maybe the updating needs a touch-up too?

**Comment by vitorgalvao** [comment]
Those instructions can admittedly be a bit hard to follow, indeed. So, what’s probably missing are the [getting set up to contribute](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#getting-set-up-to-contribute) steps. Only after you do those can you continue with updating the cask.

**Comment by danielb2** [comment]
Easy to follow. My bad, I didn't see the previous section. I need to slow down :/

**Comment by vitorgalvao** [comment]
No worries. In `cask-repair` I even now suggest that forgetting to fork might be the issue, so that benefits everyone. Happy contributing!
